spec/spec_helper.cr

```diff
@@ -48,4 +48,8 @@ Lucky::ForceSSLHandler.configure do |settings|
   settings.enabled = true
 end
 
+LuckyCache.configure do |settings|
```

Comment:
I'm not a huge fan of this but I'm not sure how best to configure the storage at the individual spec level.